src/world/sprite.h

```diff
@@ -201,7 +201,7 @@ typedef struct rct_money_effect {
 	uint8 linked_list_type_offset;	// 0x08 Valid values are SPRITE_LINKEDLIST_OFFSET_...
 	uint8 var_09;
 	uint16 sprite_index;			// 0x0A
-	uint16 flags;			// 0x0C
+	uint16 flags;					// 0x0C (Flags on money_effects do not seem to be used, so it is used for string_id)
```

Comment:
@IntelOrca  I have since switched it to the flags idea you proposed, could you tell me how you feel on this? https://github.com/OpenRCT2/OpenRCT2/pull/4825/commits/0d463d5dc73b23f7026a3252742a327d77b57563